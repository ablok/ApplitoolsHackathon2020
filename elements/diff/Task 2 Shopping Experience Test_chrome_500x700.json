[
  {
    ".grid_item": [
      {
        "op": "add",
        "path": [2],
        "val": {
          "visibility": true,
          "location": {
            "x": 15,
            "y": 779.796875
          },
          "size": {
            "width": 223,
            "height": 331
          },
          "css": {
            "display": "block",
            "backgroundColor": "rgba(0, 0, 0, 0)",
            "backgroundImage": "none",
            "textDecoration": "none solid rgb(68, 68, 68)",
            "color": "rgb(68, 68, 68)",
            "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
          }
        }
      }
    ]
  }
]
