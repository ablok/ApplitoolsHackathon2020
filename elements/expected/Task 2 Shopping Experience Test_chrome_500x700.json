[
  {
    ".grid_item": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 423.296875
        },
        "size": {
          "width": 223,
          "height": 331
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      },
      {
        "visibility": true,
        "location": {
          "x": 247.5,
          "y": 423.296875
        },
        "size": {
          "width": 223,
          "height": 331
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  }
]
